roles/foreman_repositories/tasks/_koji_staging_repos.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: 'Foreman {{ foreman_repositories_version }} Koji repository'
+  yum_repository:
+    name: foreman-koji
+    description: "Foreman {{ foreman_repositories_version }} Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-{{ foreman_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
+    priority: "1"
+    gpgcheck: no
+  tags:
+    - packages
+
+- name: 'Foreman {{ foreman_repositories_version }} Plugins Koji repository'
+  yum_repository:
+    name: foreman-plugins-koji
+    state: "{{ foreman_repositories_plugins | ternary('present', 'absent') }}"
+    description: "Foreman Plugins {{ foreman_repositories_version }} Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-plugins-{{ foreman_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
+    priority: "1"
+    gpgcheck: no
+  tags:
+    - packages
+
+- name: 'Set module_hotfixes=1'
+  lineinfile:
+    path: "/etc/yum.repos.d/{{ item }}.repo"
+    line: module_hotfixes=1
+  loop:
+    - foreman-koji
+    - foreman-plugins-koji
+  tags:
+    - packages
+  when:
+    - foreman_repositories_version != 'nightly'
+    - foreman_repositories_version is version('3.1', '<=')
```

Comment:
I think this can be safely be dropped now, we don't support 3.1 (or upgrades from it)